dataplane/apigen/protogen/protogen.go

```diff
@@ -494,6 +494,7 @@ type saiTypeInfo struct {
 	Repeated   bool
 	ProtoType  string
 	MessageDef string
+	Required   bool
```

Comment:
Does it mean a required protobuf field?

Reply:
it means it shouldn't have the `optional` keyword